include/GafferImageUI/ImageView.h

```diff
@@ -108,7 +114,9 @@ class ImageView : public GafferUI::View
 		static void registerDisplayTransform( const std::string &name, DisplayTransformCreator creator );
 		static void registeredDisplayTransforms( std::vector<std::string> &names );
 
-	protected :
+        ImageViewSignal &zoomLevelChangedSignal();
```

Comment:
Let's start with the public API - I think there should only be a `zoomLevelPlug()`. Given that, is it perhaps a case of instead blocking your connection to `cameraChangedSignal()` when you change the camera programmatically?